dask/dask-schema.yaml

```diff
@@ -1,4 +1,69 @@
 properties:
+  array_optimize:
+    type: object
+    description: |
+      ...
+
+  array_plugins:
+    type: object
+    description: |
+      ...
```

Comment:
Wondering if config variables like this belong nested in the `array` properties? Without a redirect that would be a breaking change so happy to consider that for a follow up